meetupcom_to_pyvocz.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import json
+import unicodedata
+import re
+import textwrap
+from urllib.request import urlopen
+from jinja2 import Template
+from datetime import datetime
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+
+DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def get_meetup_data(group, meetup_id):
+    # Get available data about meetup
+    url = 'https://api.meetup.com/{}/events/{}'.format(group, meetup_id)
+    json_data = urlopen(url).read()
+    return json.loads(json_data)
+
+
+def process_start(timestamp):
+    # Use only first 10 digits from timestamp and convert it to string
+    timestamp = int(str(timestamp)[:10])
+    start = datetime.fromtimestamp(timestamp)
+    return start.strftime('%Y-%m-%d %H:%M:%S')
+
+
+def process_description(description):
+    # Process description from HTML to intended YAML
+    output = []
+    description = re.sub('</p>', '\n\n', description)
+    description = re.sub('<[^<]+?>', '', description)
+    parts = description.splitlines()
+    indent = ' ' * 4
+    for part in parts:
+        part = part.strip()
+        part = textwrap.fill(part, width=75,
+                             subsequent_indent=indent,
+                             initial_indent=indent)
+        output.append(part)
+
+    del output[-1]
+
+    return '\n'.join(output)
+
+
+def load_and_fill_template(**kwargs):
+    # Load YAML template and fill it with content of **kwargs
+    template_file = os.path.join(DIR, 'event.yaml.tpl')
```

Comment:
I guess it's because writing not-so-ugly YAML is a bit tricky. But it's doable :) https://stackoverflow.com/questions/8640959/how-can-i-control-what-scalar-form-pyyaml-uses-for-my-data

Reply:
I don't know pyyaml :smile: and it seems that dump of a dictionary to YAML have keys in different order. http://pyyaml.org/wiki/PyYAML#Documentation

But yes, the true reason is my limited knowledge.

Reply:
The `dict` keys should stay in creation-order in Python 3.6+ or by using `OrderedDict` instead.

Reply:
And also, you want to preserve the order because:
- You want all events to have the same-looking YAML files
- When you update some event, you want the commit to contain only desired change